### ISSUE TITLE
" cannot import name 'BiopythonWarning' from 'Bio' "

### ISSUE THREAD

**Issue by BastianJKF** [biopython/biopython#2631]
Hi there I´m just starting using Biopython and just want to start the tutorial provided on the website... this happens just in the first command line

### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

```
>>> import sys; print(sys.version)
3.7.4 (default, Aug  9 2019, 18:34:13) [MSC v.1915 64 bit (AMD64)]
>>> import platform; print(platform.python_implementation()); print(platform.platform())
CPython
Windows-10-10.0.18362-SP0
>>> import Bio; print(Bio.__version__) 
```

(*Please copy and run the above in your Python, and copy-and-paste the output*)

### Expected behaviour

```
importing SeqIO
```

### Actual behaviour

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-5d24a7c49c42> in <module>
----> 1 from Bio import SeqIO

~\Anaconda3\lib\Bio\SeqIO\__init__.py in <module>
    388 from Bio.File import as_handle
    389 from Bio.SeqRecord import SeqRecord
--> 390 from Bio.Align import MultipleSeqAlignment
    391 from Bio.Alphabet import Alphabet, AlphabetEncoder, _get_base_alphabet
    392 

~\Anaconda3\lib\Bio\Align\__init__.py in <module>
     18 
     19 from Bio._py3k import raise_from
---> 20 from Bio.Seq import Seq
     21 from Bio.SeqRecord import SeqRecord, _RestrictedDict
     22 from Bio import Alphabet

~\Anaconda3\lib\Bio\Seq.py in <module>
     38 from Bio._py3k import basestring
     39 
---> 40 from Bio import BiopythonWarning
     41 from Bio import Alphabet
     42 from Bio.Alphabet import IUPAC

ImportError: cannot import name 'BiopythonWarning' from 'Bio' (C:\Users\49176\Anaconda3\lib\Bio\__init__.py)
```

### Steps to reproduce

```
from Bio import SeqIO
```

**Comment by peterjc** [comment]
This is strange - could you fill in the missing key information about which version of Biopython you have installed?

It looks like it was installed with conda, did you do "conda install biopython"?

**Comment by BastianJKF** [comment]
Hey there =)

Thanks for the answer. i think i tried both... however, i fond the problem. somehow i had 2 different versions of python installed. so i had to completely deinstall everything and reinstall it. now it´s working

best,
bastian

**Comment by peterjc** [comment]
Ah - glad you solved it. That situation can be very confusing.

**Comment by matteoferla** [comment]
Actually. I encountered this bug too on a fresh install.

I am running Python 3.7.6 —cannot roll over to 3.8 due to pymol and pyrosetta— with fresh miniconda3 on MacOS Catalina (10.15).

I installed it with `pip install biopython` because conda sometimes gives that annoying lowercase `import bio` business, but I got the same error as OP. `Bio.__version__` is not an extant attribute, but the wheel used was `biopython-1.76-cp37-cp37m-macosx_10_6_intel.whl`.

So I uninstalled it and tried  `conda install biopython` and worked fine. `Bio.__version__` is '1.76'.
